src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);

```

Comment:
Think about which variables should be local in the method and which should be class-level.